src/components/toogle-switch/toggle-switch.tsx

```diff
@@ -71,9 +71,10 @@ const Slider = styled.span`
 export const ToggleSwitch = ({
   checked,
   onChange,
-  name = "",
-  id = "",
-  title = "",
+  name = '',
+  id = '',
+  sliderId = '',
```

Comment:
I don't like this new prop. It is added only for testing that is not good. You can use more difficult selector instead:
`#[parentTogleSwitchId]>span`